pallets/stateful-storage/src/types.rs

```diff
@@ -85,6 +85,7 @@ pub enum PageError {
 	PageSizeOverflow,
 }
 
+/// Warning: This struct is `deprecated`. please use `ItemizedSignaturePayloadV2` instead
```

Comment:
Looks like Rust has a tag for `deprecated`:
https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute

Reply:
I'll add to extrinsic but adding it on top of the struct generated warning messages that are not supressed by 
`#[allow(deprecated)]`

Reply:
actually that is expected based on what deprecated does
`The deprecated attribute marks an item as deprecated. rustc will issue warnings on usage of #[deprecated] items. rustdoc will show item deprecation, including the since version and note, if available.`